examples/data_pipeline/transforms.py

```diff
@@ -87,16 +87,27 @@ def __init__(self, vocab):
         super(PyTextVocabTransform, self).__init__()
         self.vocab = vocab
 
+    def forward(self, tokens):
+        return [self.vocab.idx[token] if token in self.vocab.idx.keys() else 0 for token in tokens]
+
+
+class PyTextScriptVocabTransform(nn.Module):
```

Comment:
What's the difference for `PyTextScriptVocabTransform` and `VocabTransform`?

Reply:
The `VocabTransform ` calls  `self.vocab.lookup_indices(tokens)` but the `PyTextScriptVocabTransform` calls  `self.vocab.lookup_indices_1d(tokens)`. The function names are different between the two